sections/design/why-react.tex

```diff
@@ -0,0 +1,12 @@
+\section{Why React}
+For our frontend, we decided to use React.
+React is a JavaScript library for creating web applications\cite{ReactJS}. 
+\\
+React is one of the most popular choices when it comes to frontend development\cite{NPMVueReactAngular}.
+It is created by Facebook and used for their website as well as numerous other websites\cite{ReactHistory}.
+React is lightweight and allows the user to choose what additional libraries they would want to use in their project, which keeps its size down.
+\\
+It also has an extensive ecosystem that offers various solutions one could need in their web applications.
+Being this popular results in a very robust ecosystem with lots of users, but this is also needed as the libraries are managed by the react community and is not a part of the core React library, unlike other frontend libraries\cite{Vue}.
```

Comment:
sammenhængen imellem den her sætning og den forrige er ikke helt klar. Så jeg bliver lidt i tvivl om du mener med at selve react librariet bliver managed by the react community, eller om du snakker om 3rd party libraries.